superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -163,31 +177,70 @@ function ListViewCard({
             <GradientContainer>
               <CardCoverImg
                 src={imgURL}
-                onError={e => {
-                  e.currentTarget.src = imgFallbackURL;
-                }}
+                fallback={imgFallbackURL}
+                isLoading={loading}
               />
             </GradientContainer>
           </a>
           <CoverFooter className="cover-footer">
-            {coverLeft && <CoverFooterLeft>{coverLeft}</CoverFooterLeft>}
-            {coverRight && <CoverFooterRight>{coverRight}</CoverFooterRight>}
+            {!loading && coverLeft && (
+              <CoverFooterLeft>{coverLeft}</CoverFooterLeft>
+            )}
+            {!loading && coverRight && (
+              <CoverFooterRight>{coverRight}</CoverFooterRight>
+            )}
           </CoverFooter>
         </Cover>
       }
     >
-      <Card.Meta
-        title={
-          <>
-            <TitleContainer>
-              <TitleLink href={url}>{title}</TitleLink>
-              {titleRight && <div className="title-right"> {titleRight}</div>}
-              <div className="card-actions">{actions}</div>
-            </TitleContainer>
-          </>
-        }
-        description={description}
-      />
+      {loading && (
+        <Card.Meta
+          title={
+            <>
+              <TitleContainer>
+                <Skeleton.Input
+                  active
+                  css={css`
+                    width: 250;
```

Comment:
+`px`?
Also, again not sure if this big value should be a `%` or `vw` value rather than hard-coded

Last but not least though, I'm wondering if these `css` calls are somehow memoized to use the same class, or if it's generating a bunch of them here, meaning we should do something outside the function. Hopefully this is fine, but maybe worth checking for a performance hit.

Reply:
I'm pretty sure they're statically extracted and replaced with classnames, setting this up required changes to babel which would indicate static code analysis/transformation of some sort. 

Also, emotion is advertised as a performant css-in-js solution, so I'd expect them to have heavily optimized this use case.

Reply:
here's some literature around this, there's a section where they mention that by default everything is extracted into static styles using css vars where needed. 

https://medium.com/@tkh44/emotion-ad1c45c6d28b